bin/steps/conda_compile

```diff
@@ -23,16 +24,21 @@ if [ ! -d /app/.heroku/miniconda ]; then
         cp .condarc /app/.heroku/miniconda
     fi
 
+    echo "nomkl" > $HOME/.heroku/miniconda/conda-meta/pinned
```

Comment:
This now runs _only_ if there's no `.heroku/miniconda` directory already. Are you sure that's OK?

Reply:
I'm certain that's OK because this block contains setting up the `miniconda` conda environment, and if the directory exists, it means the environment is setup already, and that the file `pinned` exists with the packages here.

If it doesn't exist, then the directory `.heroku/miniconda/conda-meta` wouldn't be there either.

Am I missing something ? Or does this make sense ?

Reply:
Your explanation makes sense and if you're sure there aren't any edge cases that the old code was designed to deal with then moving the block sounds good to me.